stdlib/contrib/sranka/teams/teams.flux

```diff
@@ -16,37 +17,46 @@ message = (url, title, text, summary="") => {
     headers = {
         "Content-Type": "application/json; charset=utf-8",
     }
+
     // see https://docs.microsoft.com/en-us/outlook/actionable-messages/message-card-reference#card-fields
     // using string body, object cannot be used because '@' is an illegal character in the object property key
-    summary2 = if summary == "" 
-        then text 
-        else summary
-    shortSummary = if strings.strlen(v: summary2) > summaryCutoff 
-        then "${strings.substring(v: summary2, start: 0, end: summaryCutoff)}..."
-        else summary2
+    summary2 = if summary == "" then
+        text
+else
```

Comment:
Is this expected behavior for the formatter? I don't think we should be completely un-indenting `else` statements like this.

Reply:
That is correct. we have an issue already created for fixing that. #3656

Reply:
Ah, perfect. Thanks for the clarification :+1: